### PR TITLE
Removes all bestiality fetish material from the /tg/station codebase.

### PR DESCRIPTION
**Pull request by Iamgoofball**
We're done with this.

This is just thinly veiled bestiality fetish material. It's furry trash.

This is about as morally reprehensible as making breastfeeding videos with your kids on youtube to get around child pornography laws.

Get out of my codebase.
# daily reminder https://en.wikipedia.org/wiki/Zoophilia_and_the_law_in_the_United_States

### CODE REVIEW

**Comment by tortellinitony** [comment]
Thank you

**Comment by Bawhoppen** [comment]
goof needs to post some more pictures of himself

**Comment by Iamgoofball** [comment]
ok @Bawhoppen let me find somee

**Comment by sawrge** [comment]
here we go again

**Comment by Iamgoofball** [comment]
fuck

**Comment by PKPenguin321** [comment]
:tada:

**Comment by Qbopper** [comment]
admin purrbationed me pls remove

**Comment by Iamgoofball** [comment]
# daily reminder https://en.wikipedia.org/wiki/Zoophilia_and_the_law_in_the_United_States

**Comment by ShadowDeath6** [comment]
just make purrbation gib you and spawn a simple animal cat named "valid" that also gibs after 10 seconds

**Comment by MMMiracles** [comment]
i don't exactly like the content either but using zoophilia and bestiality as an excuse is really retarded, considering they're not even related to the removed features.

**Comment by Sligneris** [comment]
Nyaa~

**Comment by obscolene** [comment]
# hey @Iamgoofball, why do you have a picture of a half-naked child saved to your harddrive? are you a fucking pedophile or something? you literal scumbag. i hope you get fucking killed by the FBI for having literal child pornography on your computer. you are literally worse than hitler. having a stock photo of a crying child saved for image macro shitposting is LITERALLY borderline pedophillia.
# daily reminder
# https://en.wikipedia.org/wiki/Goofball_is_a_fucking_idiot

**Comment by Iamgoofball** [comment]
# Actually I'm hotlinking to various parenting guides and wikimedia websites, get destroyed @obscolene

**Comment by ChangelingRain** [comment]
ITT: goof uses something as an argument that does not actually support his argument

**Comment by Magikarpet** [comment]
someone let goofball out of his pen again. Can someone wrangle him back in so we can clean the shit?


real talk though as much as i dislike a lot of the recent cat memes you understand that shitty players will be shitty regardless of if they have fake cat ears and a buttplug surgically attached to their ass or not

**Comment by Magikarpet** [comment]
like

the pixels literally have no impact past making you autistically flustered goofball

**Comment by Iamgoofball** [comment]
@Magikarpet If you think this is about players being shitty/not shitty you're incorrect, read the OP

**Comment by Magikarpet** [comment]
do you legitimately believe that cat ears and tails on a player model encourages beastiality while being as awful as psuedo cp

**Comment by Magikarpet** [comment]
this ignores that the entire basis for the cat ears and tails are a thing based off of anime rather than actual furry conventions or standards

**Comment by optimumtact** [comment]
@obscolene don't drag yourself down to goofs level, he'll win on experience

**Comment by Iamgoofball** [comment]
oranges is right

**Comment by Magikarpet** [comment]
like you couldve raised the point that the content is borderline and asked for polls to remove them

instead you took the most retarded approach and now you've gained support for the side you opposed

any rational person looking at this will see that goofball equates allowing catpeople to be equivalent to child pornography or beastiality

why do you do this goof do you need me to hire a therapist

are you pretending to be retardewd

am i the most retarded person here for actually replying to this half baked pr

**Comment by Iamgoofball** [comment]
>like you couldve raised the point that the content is borderline and asked for polls to remove them

as has been stated we've tried this by bringing up it's garbage fetish material

\>ask poll to be ran to remove fetish material that multiple admins indulge in
\>expect it to be taken seriously/not immediately undermined by admins

**Comment by Iamgoofball** [comment]
@YakumoChen oh no you posted an anime what am i going to do

**Comment by ChangelingRain** [comment]
should we even bother pretending this is going to be merged?

**Comment by kevinz000** [comment]
nah, lets be nice and spare goofball the effort!

**Comment by kevinz000** [comment]
wait, did I say effort...?

**Comment by ChangelingRain** [comment]
the answer is no. it's always no, when goof opens a meme pr that doesn't compile.

**Comment by MrPerson** [comment]
Who needs compiling when there's a message to be sent?

**Comment by Iamgoofball** [comment]
@ChangelingRain it's almost as if I was fixing the error

**Comment by OneOneThreeEight** [comment]
free this pr
free goof's genius

**Comment by TheFriggenSawbones** [comment]
@Iamgoofball hi goof whats up

**Comment by Iamgoofball** [comment]
hi

**Comment by TheFriggenSawbones** [comment]
how're you

**Comment by dannno** [comment]
are you just joking or are you actually insane

**Comment by KorPhaeron** [comment]
24 hour rule

**Comment by kevinz000** [comment]
=|

**Comment by Tacolizard** [comment]
'goof is a net positive to the codebase'

File: code/modules/mob/living/carbon/human/species_types/humans.dm [hunk 1]
```diff
@@ -3,29 +3,15 @@
 	id = "human"
 	default_color = "FFFFFF"
 	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS)
-	mutant_bodyparts = list("tail_human", "ears", "wings")
-	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	mutant_bodyparts = list("wings")
```
Comment (Tokiko1) [inline code/modules/mob/living/carbon/human/species_types/humans.dm]: Harpies when?

**Comment by Militaires** [comment]
very good

**Comment by fludd12** [comment]
>removes cat ears as supposed bestiality material

>adds wings instead

secretly, goof just wanted everyone to like his feather-furry deviantart OC

**Comment by Steelpoint** [comment]
I'm very content if we remove cat people. Absolutly no effort content that should never have been made a actual species people can play.

**Comment by bgobandit** [comment]
why is obscolene so upset about removing kitty ears
makes you think

**Comment by Militaires** [comment]
@fludd12 he's not adding wings

**Comment by Repukan** [comment]
Catgirls are old tired shit.

Now harpies on the other hand...

**Comment by fludd12** [comment]
@Militaires i was almost funny and now i'm not.
now i'm just sad.

**Comment by Militaires** [comment]
@fludd12 now u are again

**Comment by bunny232** [comment]
hagraven
ATLEAST WINGS ARE BETTER THEN CAT SHITS

**Comment by obscolene** [comment]
@bgobandit even if i cared about cats being removed, i wouldn't have to worry because this is a certified goof pr TM

**Comment by Wutnold124** [comment]
goon you're my favorite person in the world

**Comment by RickDipper** [comment]
Remove the shit, it's all a "le joke" but actual ERP and Furry shit has crept onto the server.

**Comment by Pascal125** [comment]
Why is this still open.

**Comment by improvedname** [comment]
Goofball is anger because he got burger king from his mom but he wanted mcdonalds

File: code/modules/mob/living/carbon/human/species_types/humans.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species_types/humans.dm]: Wings are for that lavaland potion you retards

**Comment by Iamgoofball** [comment]
Wings are for that lavaland potion you retards, the one that lets you ignore chasms

**Comment by Shihrt** [comment]
>goof still at it

crying-babies

**Comment by Iamgoofball** [comment]
I've been memed

**Comment by optimumtact** [comment]
@RemieRichards

**Comment by RemieRichards** [comment]
@optimumtact The rule is about being fair to legitimate PRs.
No goof PRs are legitimate.

**Comment by optimumtact** [comment]
I don't think you should overrule our design lead, or make light of a rule that MSO has asked us to enforce in the past

**Comment by RemieRichards** [comment]
I'm closing this PR when it gets to 24hrs regardless, so there's literally no point keeping it open in a limbo where it:
A. Cannot be merged before 24hrs
B. Won't be merged after 24hrs

**Comment by GeneralGoose** [comment]
horay 3rd anti-furfag pr closed congrats TG 
my analysis of your repo shows you might want to have some features from vore ported :^)

**Comment by Bawhoppen** [comment]
working on that one no worries

**Comment by GeneralGoose** [comment]
goof remove all furries and you get a nobel prize

**Comment by Steelpoint** [comment]
Goof might be insane but I do think cat people are a bit too far a addition to /tg/. 

Locking it to genetics is one thing, making it a round start race is another and I wholeheartedly think it should be removed.

**Comment by Kraseo** [comment]
hey mr moral against zoophilia man you forgot lizardmen and flymen

**Comment by Steelpoint** [comment]
Difference is Lizardmen and Flymen both look uniquely non-human and have a somewhat extensive lore background on them (more for lizardmen than flymen).

Catpeople are just Humans with cat ears and tails.

**Comment by Bawhoppen** [comment]
or we can just have a game that's interesting and full of variety with stuff like that

**Comment by Bawhoppen** [comment]
24hr rule for principle is fine

**Comment by FantasticFwoosh** [comment]
Focus less on goof, does anybody actually want to see this feature go? I don't see any actual real opposal to the feature going as much as indifference outweighed by the people wanting to go so i think we should give this PR a chance.

> Now talking on a serious basis, catpeople are often shown to have overblown exagerrated and self referencial ego's like Peter Vandalmar's gross male Neko impression. A tail on a sprite causes players to enter a different mindset isn't good roleplay (which we hardly prize) or gameplay.

**Comment by Crexfu** [comment]
Personally, I would rather have the option then not have the option.

**Comment by lzimann** [comment]
Yes. I want this to go.

**Comment by RemieRichards** [comment]
@Leoz, Don't fight me on this, we're not accepting this PR, 24hr or no.

**Comment by lzimann** [comment]
I will fight, also 24h etc

**Comment by RemieRichards** [comment]
I'll see you in 16 hours then.

**Comment by kevinz000** [comment]
>still enabling goof

come on lzimann hating catears or not

**Comment by Earthykiller127** [comment]
i have never seen people pull this big of a debate for meaningless furry shit in autism in space 15 simulator

**Comment by kevinz000** [comment]
it's just cosmetics lmao this is gold

**Comment by optimumtact** [comment]
it's not though kevin, it stands for quite a bit more in the minds of everyone here and you know that

**Comment by KorPhaeron** [comment]
http://i.imgur.com/TqGY2Xn.png

Convince the headmins to turn it off if you dislike it, there is already a config

**Comment by KorPhaeron** [comment]
@Cheridan

**Comment by lzimann** [comment]
Having a config still leaves this in the code. Just because something is a config doesn't mean it should stay forever. Game modes are a config, yet we still remove them from time to time. Linking a vote is stupid, we can't use the "democracy" argument for something and completely ignore for other things.

**Comment by RemieRichards** [comment]
image

**Comment by lzimann** [comment]
The thing is, I am being the change I want to see.

**Comment by RemieRichards** [comment]
You can keep your pr (pointlessly) open until 24hr (where I will close it immediately), but we're not having shitposting on it.
Either it's closed and can be shitposted on, or open and not.
Your choice.

**Comment by KorPhaeron** [comment]
Get elected headmin if you want to remove this

**Comment by lzimann** [comment]
Hmmm

**Comment by KorPhaeron** [comment]
I just wanted to watch goof shitpost more but I've grown tired of the joke

**Comment by Iamgoofball** [comment]
It's hard to reply to shitpost when remie locks my PR

@KorPhaeron ok so I guess I better add ERP code with full vore to the game it's ok it's config optioned

**Comment by RemieRichards** [comment]
I locked it to prevent shitposts.
your posts are also shitposts.

**Comment by Iamgoofball** [comment]
No, you locked it to plug your ears and go LALALA I CAN'T HEAR OPINIONS I DISAGREE WITH

**Comment by sawrge** [comment]
Isn't that exactly what you're doing with this PR goof

**Comment by Iamgoofball** [comment]
I at least let them reply and I respond to their statements

**Comment by RemieRichards** [comment]
Says the guy making the PR that goes against what's been voted. (and who's own pr vote is going against him, but I'm sure we'll see the votes miraculously flip considering you have an army of bots to vote for you, it's happened before)

And when all the posts are "LMAO FURRIES", "THE CAT METAGANGS TOUCHED ME", "YOU CANT HAVE 10 PIXELS ON YOUR HEAD THAT'S AN INSULT, 10 PIXELS KILLED MY GRAN" I consider that shitposting that requires containment.

**Comment by RemieRichards** [comment]
@Iamgoofball also I'm pretty sure pr authors can still comment, unless that's changed.

**Comment by Wyzack** [comment]
OH X IS OKAY I GUESS I WILL JUST DO Y THEN BECAUSE THEY ARE TOTALLY THE SAME TY SHITE ADMINS

**Comment by lzimann** [comment]
Stop using vote as an argument. We only pretend to care about voting when it's in favour of what we like. Let's are least stop pretending that.

**Comment by ohnopigeons** [comment]
It should definitely be a factor to be considered.

**Comment by KorPhaeron** [comment]
There are times when its necessary to do an unpopular thing for long term good but basically what ohnopigeons said, we should still be aiming to make people happy in the end.

**Comment by Iamgoofball** [comment]
> also I'm pretty sure pr authors can still comment, unless that's changed

This has never been the case.

**Comment by Iamgoofball** [comment]
> but I'm sure we'll see the votes miraculously flip considering you have an army of bots to vote for you, it's happened before

1. Not a democracy 

2. Lmao where's proof memelord

**Comment by The-clown-known-as-Pepe** [comment]
okay goof, we get you get bullied at school sometimes but you can't just take it out on code.

**Comment by Ausops** [comment]
The official headmin position probably will be your a bunch of retards for falling for goof's latest bait meme pr

I will ignore it and advise my colleagues to ignore it as the separation between codebase and server is absolute. We send our moral support, express our vague concerns, and wish to make it clear that we diplomatically are rooting for you, Cheridan, and hope you resolve this crisis cleanly and quickly by asserting your cherry-flavoured dominance all over goof.

**Comment by Iamgoofball** [comment]
This wasn't a meme PR

**Comment by PKPenguin321** [comment]
Can we at least remove cat ears from kevinz specifically

**Comment by ShadowDeath6** [comment]
I don't like cat parts on people but now that you can surgically remove them I think it'll be a bit better than it was before

**Comment by OneOneThreeEight** [comment]
#FreeGOOF

**Comment by GeneralGoose** [comment]
time for tg to split into paradise-vore union and gooftopia

**Comment by Iamgoofball** [comment]
new goofball republic

**Comment by Difrit** [comment]
Get in the bin

**Comment by ZomgPonies** [comment]
We don't want him at Paradise, he's repo-banned.

**Comment by Repukan** [comment]
You know if he's banned in Paradise we're obviously doing something right.

**Comment by Iamgoofball** [comment]
Because apparently swear words in reviews is a no no.

**Comment by Core0verload** [comment]
@Iamgoofball you tried. Try lizards next time.

**Comment by ThatOneGuy42** [comment]
THIS IS A WORK OF ART CONTINUE YOU MAD MAN MAKE GOD AND HIS ANGELS PROUD

**Comment by GeneralGoose** [comment]
dont give up goof i believe in you

**Comment by MattDuskson** [comment]
Cleanse the impure!

**Comment by Spacedong** [comment]
you're all fucking faggots go goof remove furries

**Comment by ohnopigeons** [comment]
reddit pls go

**Pull request closed without merging**